main.py

```diff
@@ -33,6 +33,7 @@
     parser.add_argument("-v", "--verbose", action="store_true")
     parser.add_argument("-l", "--logprobs", action="store_true")
     parser.add_argument("--upload_url", type=str)
+    parser.add_argument("--run_name", type=str, required=False)
```

Comment:
added a param to optionally add a run name when we are running an eval

this run name is then stored as a static JSON object, and can be used by the eval visualizer

Reply:
Done, thank you for pointing this out!